src/run_verification_case.py

```diff
@@ -38,6 +38,9 @@ def run_verification_case(item_dict, run_path_postfix=""):
         if ("parameters" in item.item["datapoints_source"])
         else None
     )
+    cls.points = list(
```

Comment:
we use the points field as a guard to make sure verification case with data contains all points verification class needs. This is checked in constructor of CheckLibBase. Assigning points value here partially voided this guard. For instance, when the verification case does not specify all points needed for the verification, there is no guard to warn this.

Reply:
It makes sense to me and I changed this part based on our discussion Jerry. I'd appreciate it if you could review this one more time.